content/questions/actions/question-093.md

```diff
@@ -0,0 +1,13 @@
+---
+archetype: "questions"
+title: "Question 093"
+question: "Which of the following is true about self-hosted runners in GitHub Actions?"
+draft: false
+---
+
+
+> https://docs.github.com/en/actions/hosting-your-own-runners/managing-self-hosted-runners/about-self-hosted-runners 
+1. [ ] They are free for all repositories
+1. [x] They can run on any operating system
```

Comment:
Don't think that;s true - there is a list [here](https://docs.github.com/en/actions/hosting-your-own-runners/managing-self-hosted-runners/about-self-hosted-runners#supported-architectures-and-operating-systems-for-self-hosted-runners)

Reply:
The intention was that self-hosted runners don't restrict users like GitHub hosted runners, but you have a point. "ANY operating systems" wasn't a reasonable answer, so let me exclude that one as well